theme/src/constants/toc.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Enum values for identifying header levels in the global table of contents
+ * data structure.
+ *
+ * TODO Come up with better names for header levels.
```

Comment:
<img width="600" alt="Screen Shot 2021-09-14 at 4 21 32 PM" src="https://user-images.githubusercontent.com/2176050/133346468-155fd591-5879-443f-aaa5-88be2edb2b81.png">

Any opinions on the [naming](https://hilton.org.uk/blog/why-naming-things-is-hard) for the different levels? I feel like there's a better way to name this, but I can't think of alternatives 😆

Reply:
Maybe:

1. Section
2. Subsection
3. Item

Or

1. Category
2. Section
3. Subsection

Reply:
Suggesting this but not sure it makes sense so just throwing it out there...
1. Page
2. Title
3. Section

Reply:
honestly I'd find it more clear and more extendable if we simply named it heading level n

Reply:
> honestly I'd find it more clear and more extendable if we simply named it heading level n

lmao I thought about this as well, but then I wasn't sure what to name the `CategoryTableOfContents` component too 😆 But I like the `Level<n>` naming. I'll refactor the code to use that naming convention